blueprints/addon/files/addon-config/ember-try.js

```diff
@@ -19,49 +19,25 @@ module.exports = {
     },
     {
       name: 'ember-release',
-      bower: {
-        dependencies: {
-          'ember': 'components/ember#release'
-        },
-        resolutions: {
-          'ember': 'release'
-        }
-      },
       npm: {
         devDependencies: {
-          'ember-source': null
+          'ember-source': 'latest'
```

Comment:
This is slightly different from what we had before.  `latest` is going to be the most recent _tagged_ release on NPM, whereas `release` in bower was the build from the last commit on the `release` branch.

I believe that we want all three of these to be something like:

```js
npm: {
  devDependencies: {
    'ember-source': 'github:emberjs/ember.js#release',
  }
}
```

Reply:
I will update all the branch hashes, as for github:, that can be left off and it will be implied. Would you prefer explicit or implicit github?

Reply:
I like being explicit, mostly just because I've had the question asked by _a lot_ of teammates "what does this mean, where is this version number from" when using without `github:` prefix.  It _probably_ doesn't matter here (addon authors are almost certainly going to be able to figure it out), but if its all the same to you I like the prefix...

Reply:
will do!